### PR TITLE
setting django-admin

### PR DESCRIPTION
**Pull request by DzmitrySalanovich**

### CODE REVIEW

File: products/admin.py [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+    pass
+admin.site.register(Category, CategoryAdmin)
+
+class ProductAdmin(admin.ModelAdmin):
+    pass
+admin.site.register(Product, ProductAdmin)
+
+class Product_attAdmin(admin.ModelAdmin):
+    pass
+admin.site.register(Product_att, Product_attAdmin)
```
Comment (duckyou) [inline products/admin.py]: Could you please refactor this file using PEP8 principles.

Thanks.

File: authentication/admin.py [hunk 2]
```diff
@@ -1,3 +1,7 @@
 from django.contrib import admin
+from authentication.models import User, UserManager
+
+class UserAdmin(admin.ModelAdmin):
+    pass
+admin.site.register(User, UserAdmin)
```
Comment (duckyou) [inline authentication/admin.py]: PEP8, pls

**Review by duckyou — CHANGES_REQUESTED** [review]
Could you please also create `.env` file example, please... :smiley:

File: authentication/admin.py (hunk 2 above)
Comment (MaksimLion) [reply authentication/admin.py]: you can use decorator syntax for more readable code

**Review by MaksimLion — CHANGES_REQUESTED** [review]

File: authentication/admin.py [hunk 3]
```diff
@@ -1,3 +1,8 @@
 from django.contrib import admin
+from authentication.models import User, UserManager
+
+class UserAdmin(admin.ModelAdmin):
+    pass
+
+admin.site.register(User, UserAdmin)
```
Comment (MaksimLion) [inline authentication/admin.py]: `@admin.register(User)
class UserAdmin(admin.ModelAdmin):
    pass
`
should look like this

File: products/models.py [hunk 4]
```diff
@@ -12,8 +12,8 @@ def __str__(self):
 class Product(models.Model):
     name = models.CharField(max_length=30)
     sku = models.IntegerField()
-    discount = models.PositiveIntegerField(max_length=5)
-    price = models.PositiveIntegerField(max_length=10)
+    discount = models.PositiveIntegerField()
+    price = models.PositiveIntegerField()
     quantity = models.PositiveIntegerField()
     category = models.ManyToManyField(to=Category)
```
Comment (MaksimLion) [inline products/models.py]: Add related name

File: example.env [hunk 5]
```diff
@@ -0,0 +1,7 @@
+SECRET_KEY=
+'ENGINE'
+'NAME'
+'USER'
+'PASSWORD'
+'HOST'
+'PORT'
```
Comment (MaksimLion) [inline example.env]: Is incorrect env file

**Review by MaksimLion — CHANGES_REQUESTED** [review]

**Review by duckyou — APPROVED** [review]

File: orders/admin.py [hunk 6]
```diff
@@ -1,3 +1,11 @@
 from django.contrib import admin
+from orders.models import Order, LineItem
 
-# Register your models here.
+class OrderAdmin(admin.ModelAdmin):
+    pass  
+
+class LineItemAdmin(admin.ModelAdmin):
+    pass
+
```
Comment (MaksimLion) [inline orders/admin.py]: Please use decorator instead this approach, when you use decorator, it's give more flexible code

**Review by MaksimLion — CHANGES_REQUESTED** [review]

**Review by MaksimLion — APPROVED** [review]

**Pull request merged**